cmake/templates/license.h.in

```diff
@@ -66,6 +66,37 @@ jsoncpp:
   Public Domain "license" you can re-license your copy using whatever
   license you like.
 
+scanner/token:
+  The libsolidity/parsing/{scanner,token}.{h,cpp} files are dervied from
+  code originating from the V8 project licensed under the following terms:
+
+  Copyright 2006-2012, the V8 project authors. All rights reserved.
+  Redistribution and use in source and binary forms, with or without
+  modification, are permitted provided that the following conditions are
+  met:
+
+      * Redistributions of source code must retain the above copyright
+        notice, this list of conditions and the following disclaimer.
+      * Redistributions in binary form must reproduce the above
+        copyright notice, this list of conditions and the following
+        disclaimer in the documentation and/or other materials provided
+        with the distribution.
+      * Neither the name of Google Inc. nor the names of its
```

Comment:
I wonder if putting in the license which shows the name Google in `solc --license` is considered advertising? 😕